minerl/env/core.py

```diff
@@ -312,11 +318,19 @@ def _process_observation(self, pov, info):
             info = json.loads(info)
         else:
             info = {}
-        info['pov'] = pov
+        info['pov'] = self._last_pov = pov
+
+        obs_space = deepcopy(self.observation_space.spaces)
+        obs_dict = {}
+
+        try:
+            for key in obs_space:
+                handler_fn = self.obs_handlers[key]
+                obs_dict[key] = handler_fn(info, obs_space)
+        except KeyError:
+            print(f"Could not find handler for observation space {key}, returning empty.")
```

Comment:
It looks like this causes the MineRL tests to fail on Python 3.5 because f-strings aren't supported (see the failing Travis build link on the PR); I know we're all using 3.6 or above, so it's not a huge deal, but since it's not too costly, it might be worth changing this to a `.format()`, just so we can keep those Travis test as a meaningful/non-noisy alarm signal

Reply:
Looking at the pytests that Travis runs, it seems like Travis fails sometimes when it fails to download the MineRL dataset.